### DISCUSSION TITLE
Add Input Source Controls

### DISCUSSION THREAD

**Discussion by zhmrsk** [xanderfrangos/twinkle-tray#395] [Ideas]
DDC/CI Allows you to change video signal sources. This would be a great feature in a future version of the app. This is very useful for those who connect multiple computers to multiple monitors (like me switching between Mac and PC) Thank you for your attention, I hope you understand me and implement this feature

**Comment by CorsairPass** [comment]
That would be a perfect addon in my experience.
I use my work notebook & my personal pc on both monitors.
If i could switch with a keystroke, that would be THE perfect add on to make my setup complete.

**Comment by decodering22** [comment]
Was this ever added, this is exactly what I'm looking for
